generators/generator-base-entity.cjs

```diff
@@ -17,23 +17,34 @@
  * limitations under the License.
  */
 const BaseBlueprintGenerator = require('./generator-base-blueprint');
-const { CUSTOM_PRIORITIES_ENTITIES, compat } = require('../lib/constants/priorities.cjs');
+const { CUSTOM_PRIORITIES_ENTITIES, PRIORITY_NAMES, QUEUES } = require('../lib/constants/priorities.cjs');
 
 const {
-  QUEUE_EACH_ENTITY_PRIORITY,
-  CONFIGURING_EACH_ENTITY_PRIORITY,
-  PREPARING_EACH_ENTITY_PRIORITY,
-  PREPARING_EACH_ENTITY_FIELD_PRIORITY,
-  PREPARING_EACH_ENTITY_RELATIONSHIP_PRIORITY,
-  WRITING_ENTITIES_PRIORITY,
-} = compat;
+  CONFIGURING_EACH_ENTITY,
+  LOADING_EACH_ENTITY,
+  PREPARING_EACH_ENTITY,
+  PREPARING_EACH_ENTITY_FIELD,
+  PREPARING_EACH_ENTITY_RELATIONSHIP,
+  POST_PREPARING_EACH_ENTITY,
+  WRITING_ENTITIES,
+} = PRIORITY_NAMES;
+
+const {
+  CONFIGURING_EACH_ENTITY_QUEUE,
+  LOADING_EACH_ENTITY_QUEUE,
+  PREPARING_EACH_ENTITY_QUEUE,
+  PREPARING_EACH_ENTITY_FIELD_QUEUE,
+  PREPARING_EACH_ENTITY_RELATIONSHIP_QUEUE,
+  POST_PREPARING_EACH_ENTITY_QUEUE,
+  WRITING_ENTITIES_QUEUE,
+} = QUEUES;
 
 /**
  * This is the base class for a generator that generates entities.
  */
 class JHipsterBaseEntityGenerator extends BaseBlueprintGenerator {
```

Comment:
Shouldn't it be `JHipsterBaseEntit**ies**Generator` (and the file name called base-entit**ies**?
Here we have no method that configures a single entity